sources/core/objects/mobs/Human.cpp

```diff
@@ -184,7 +184,22 @@ void Human::ProcessMessage(const Message& message)
         {
             Phrase phrase;
             phrase.from = GetName().toHtmlEscaped();
-            phrase.text = text.toHtmlEscaped();
+            phrase.text = text.toHtmlEscaped().trimmed();
+	    
+            int lastchar_position=phrase.text.length()-1;
+            QChar lastchar=phrase.text[lastchar_position];
```

Comment:
The `text` size should be checked, because there is a possibility that it will be empty and `operator[]` will crash the game.